release/karaf/distro/README_FUSE_INTEGRATION.md

```diff
@@ -11,7 +11,7 @@ Camel components used in kie: kie-camel and jbpm-workitem-camel components
 
 Requirements
 ============
-DROOLS_VERSION --> 6.2.0.Final
+DROOLS_VERSION --> ${DROOLS_VERSION}
```

Comment:
This should be I believe removed or changed to something like `DROOLS_VERSION => 6.3.0.Final`. The current form does not add any additional info.